assets/airports/mdsd.json

```diff
@@ -0,0 +1,5080 @@
+                ["SD140", "SD141", "KERSO"]
+            ]
+        },
+        "KERSO1": {
+            "icao": "KERSO1",
+            "name": "Kerso One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "_CDO008015"],
+            "exitPoints": {
+                "KERSO": ["KERSO"]
+            },
+            "draw": [
+                ["CDO", "KERSO"]
+            ]
+        },
+        "KERSO2": {
+            "icao": "KERSO2",
+            "name": "Kerso Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "CDO"],
+            "exitPoints": {
+                "KERSO": ["KERSO"]
+            },
+            "draw": []
+        },
+        "KODIX1": {
+            "icao": "KODIX1",
+            "name": "Kohdeex One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "_CDO338015"],
+            "exitPoints": {
+                "KODIX": ["KODIX"]
+            },
+            "draw": [
+                ["CDO", "PITOX", "KODIX"]
+            ]
+        },
+        "KODIX2": {
+            "icao": "KODIX2",
+            "name": "Kohdeex Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "CDO", "PITOX"],
+            "exitPoints": {
+                "KODIX": ["KODIX"]
+            },
+            "draw": []
+        },
+        "LECKY": {
+            "icao": "LECKY",
+            "name": "Lecky",
+            "suffix": {"17":"1J", "35":"1L"},
+            "rwy": {
+                "17": ["SD130", "SD131"],
+                "35": ["^SD140", "SD142", "SD143"]
+            },
+            "body": [],
+            "exitPoints": {
+                "LECKY": ["LECKY"]
+            },
+            "draw": [
+                ["SD131", "LECKY"],
+                ["SD143", "LECKY"]
+            ]
+        },
+        "LECKY1": {
+            "icao": "LECKY1",
+            "name": "Lecky One",
+            "rwy": {
+                "35": []
+            },
+            "body": [["^PITOX", "A30-"], "CDO"],
+            "exitPoints": {
+                "LECKY": ["LECKY"]
+            },
+            "draw": [
+                ["CDO", "LECKY"]
+            ]
+        },
+        "LECKY2": {
+            "icao": "LECKY2",
+            "name": "Lecky Two",
+            "rwy": {
+                "17": []
+            },
+            "body": [["^PODIS", "A30-"], "_CDO237015"],
+            "exitPoints": {
+                "LECKY": ["LECKY"]
+            },
+            "draw": []
+        },
+        "MIROX": {
+            "icao": "MIROX",
+            "name": "Mirox",
+            "suffix": {"17":"1J", "35":"1L"},
+            "rwy": {
+                "17": ["SD130", "SD131", ["SD132", "A40-"]],
+                "35": ["^SD140", "SD144", "SD145", ["SD146", "A30"], ["SD148", "A60+"]]
```

Comment:
Interestingly, the chart shows the existence of SD144, but the fix is nowhere to be found under the description of the procedure:
![chart](https://user-images.githubusercontent.com/13125311/28041404-2c431f0c-65c1-11e7-8218-423078e3c873.PNG)
![description](https://user-images.githubusercontent.com/13125311/28041407-2e51ecb0-65c1-11e7-9bc7-d1aa2c0abd14.PNG)